### PR TITLE
IAM-297: forSpec example

### PR DESCRIPTION
**Pull request by wilsoncg**
Addition of EffectiveDateRelative forSpec policy example

Developed/Tested by running the sample jupyter notebooks on my lusid test domain.

### CODE REVIEW

File: examples/entitlements/Portfolio Entitlements.ipynb [hunk 1]
```diff
@@ -2,7 +2,8 @@
  "cells": [
```
Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    def create_portfolio(Port_code,scope='',display_name='',base_currency='GBP'):
</pre><p>follow python convention for naming and give descriptive name: <code>Port_code</code>  &gt; <code>portfolio_code</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    def upsert_fx(scope, port_code, tran_date=datetime(2021, 5, 24, tzinfo=pytz.utc)):
</pre><p> </p><p><code>tran_date</code>  &gt; <code>transaction_date</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>add a descriptive message affirming they upload was successful, if you are just going to use the links wrap each call in <code>display()</code>  so you get a message per call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>This needs a description of what its doing (or link to relevant KB article), its not clear to new reader what is happening</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    def create_policy_portfolio_scope_access(scope,portfolio_code,deactivate=datetime(2021, 12, 31, tzinfo=pytz.utc), forSpecs=None, activate_delay=0):
</pre><p> </p><p><code>forSpecs</code>  &gt; <code>for_specs</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>use list comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>use list comprehension</p><p><code>IdSelector</code>  &gt; <code>id_selector</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #3.        policy_id_list=[]
</pre><p> </p><p>use list comprehension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>add a decription of what these are used for (or point to a KB article)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #10.    #source1_150_delay=create_policy_portfolio_scope_access('Source1','1250', activate_delay=14)
</pre><p> </p><p>remove  unused code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #14.    #source_1_6305_policy_id=create_policy_portfolio_scope_access('Source1','6305', activate_delay=14)
</pre><p> </p><p>remove unused code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (rizwansaeed) [inline examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>I presume this example is validated in the UI? If so add a note at the end for the reader</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (wilsoncg) [reply examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>resolved in #341</p>

Comment (wilsoncg) [reply examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>resolved in #341</p>

Comment (wilsoncg) [reply examples/entitlements/Portfolio Entitlements.ipynb:2]: <p>resolved in #341</p>

**Pull request merged**